pytest.ini

```diff
@@ -19,6 +19,7 @@ markers =
     regression: marks regression tests (deselect with '-m "not regression"')
     network: marks tests requiring network access
     slow: marks tests as slow (approx >1s) (deselect with '-m "not slow"')
+    scream_run: marks tests that require running inside scream container (delesect with '-m "not scream_run"')
```

Comment:
I think my preferred way to avoid having the scream runtime test get called with the rest of the fv3net tests would be to just add `/tests/scream_run_integration` into the `norecursedirs` list above, and then you don't need to create another type of pytest marker. As you can see we do this with a lot of tests that need special handling.

Reply:
I did not realize that those test get skipped. Moving to `norecursedirs`